app/volitile/constants.go

```diff
@@ -0,0 +1,4 @@
+package volitile
```

Comment:
Why is this package called volitile?

Not a blocker but #PUC?

Reply:
Mostly because I spelled it incorrectly. :sweat_smile: I PUC'd it up.